src/pages/services/create.tsx

```diff
@@ -22,8 +22,8 @@ const Service: NextLayoutPage = () => {
 
   return (
     <Form className="bg-cyan-700 text-white px-5 py-8 h-full" schema={ServiceSchema} onSubmit={handleSubmit}>
-      <h1 className='mb-5'>Création d'un service</h1>
-      <div className='grid grid-cols-12 space-y-3'>
+      <h1 className="mb-5">Création d&aposun service</h1>
```

Comment:
I think using template literals is better, or maybe just a js string with double quotes

 <h1 className="mb-5">"Création d'aposun service"</h1>